.github/workflows/ci.yml

```diff
@@ -183,13 +183,23 @@ jobs:
           docker compose up -d relay-1 relay-2 --no-build
           docker compose up -d gateway --no-build
           docker compose up -d client --no-build
+      - name: Add 10ms simulated latency
+        run: |
+          # Add 10ms latency to the client
+          docker compose exec -d client tc qdisc add dev eth0 root netem delay 10ms
+          # Add 10ms latency to the gateway
+          docker compose exec -d gateway tc qdisc add dev eth0 root netem delay 10ms
+          # Add 10ms latency to the relay-1
+          docker compose exec -d relay-1 tc qdisc add dev eth0 root netem delay 10ms
+          # Add 10ms latency to the relay-2
+          docker compose exec -d relay-2 tc qdisc add dev eth0 root netem delay 10ms
       - name: "Performance test: ${{ matrix.test_name }}"
         timeout-minutes: 5
         env:
           TEST_NAME: ${{ matrix.test_name }}
         run: |
           ./scripts/tests/perf/${{ matrix.test_name }}.sh
-          jq '{ "${{ matrix.test_name }}": { "throughput": { "value": .end.sum_received.bits_per_second } } }' ./${{ matrix.test_name }}.json > ./${{ matrix.test_name }}.bmf.json
+          jq '{ "${{ matrix.test_name }}": { "retransmits": { "value": .end.sum_sent.retransmits }, "throughput": { "value": .end.sum_received.bits_per_second } } }' ./${{ matrix.test_name }}.json > ./${{ matrix.test_name }}.bmf.json
```

Comment:
I am guessing `sum_sent` always refers to the sender and is independent of which direction the benchmark was executed in?

Reply:
Corrent - the field is present in both directions. UDP won't have it though, in which case this will just be `null`.